### PR TITLE
maps in the new snack vendors + beer closet

### PR DESCRIPTION
**Pull request by crunchypretzels**
As title. Adds four FrontierVend machines to the Horizon.

This PR also adds a closet of beer to the Horizon's bar backroom, filled with canned beer. Crack open a cold one with the lads after that shootout.

### CODE REVIEW

**Comment by crunchypretzels** [comment]
!review

**Comment by DreamySkrell** [comment]
image
you put the new vendor here, replacing a guest pass terminal and a potted plant
do not do that
revert that change and put the new vendor near the docks there

**Comment by DreamySkrell** [comment]
image
You put a coffee machine there. Why? There is one just below it already.

**Comment by DreamySkrell** [comment]
image
put yellow hatching under the new vendor here so it is consistent with other vendors in the area

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
as above

File: maps/sccv_horizon/sccv_horizon-2_deck_2.dmm [hunk 1]
```diff
@@ -5621,7 +5621,9 @@
 /turf/simulated/floor/plating,
 /area/horizon/security/hallway)
 "czP" = (
-/obj/structure/closet/cabinet,
+/obj/structure/closet/cabinet{
+	name = "beer cabinet"
+	},
```
Comment (SleepyGemmy) [inline maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:5626]: misplaced rename?

File: maps/sccv_horizon/sccv_horizon-2_deck_2.dmm [hunk 2]
```diff
@@ -44895,7 +44898,21 @@
...
+/obj/item/storage/box/fancy/yoke/ebisu,
+/obj/item/storage/box/fancy/yoke/ebisu,
+/obj/item/storage/box/fancy/yoke/moonlabor,
+/obj/item/storage/box/fancy/yoke/moonlabor,
+/obj/item/storage/box/fancy/yoke/beer,
+/obj/item/storage/box/fancy/yoke/beer,
+/obj/item/storage/box/fancy/yoke/shimauma,
+/obj/item/storage/box/fancy/yoke/shimauma,
+/obj/item/storage/box/fancy/yoke/whistlingforest,
+/obj/item/storage/box/fancy/yoke/whistlingforest,
```
Comment (SleepyGemmy) [inline maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:44915]: add a subtype (e.g. `/obj/structure/closet/cabinet/beer`), add the items to the subtype, and map in the subtype instead.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
hatching should be layered under the spline.

File: maps/sccv_horizon/sccv_horizon-2_deck_2.dmm (hunk 1 above)
Comment (crunchypretzels) [reply maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:5626]: misplaced rename!

File: code/game/objects/structures/crates_lockers/closets/secure/bar.dm [hunk 3]
```diff
@@ -23,3 +23,26 @@
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
+
+/obj/structure/closet/secure_closet/cabinet/beer
+	name = "beer closet"
+	desc = "A cabinet stacked end-to-end with six packs of beer."
+	req_access = list(ACCESS_BAR)
```
Comment (DreamySkrell) [inline code/game/objects/structures/crates_lockers/closets/secure/bar.dm:30]: default access should be null / no access required

File: code/game/objects/structures/crates_lockers/closets/secure/bar.dm [hunk 4]
```diff
@@ -23,3 +23,26 @@
...
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
+
+/obj/structure/closet/secure_closet/cabinet/beer/unlocked //for awaysites
```
Comment (DreamySkrell) [inline code/game/objects/structures/crates_lockers/closets/secure/bar.dm:47]: should be /horizon subtype specifically for horizon

What you did here was the standard for a long time, but I would like us from now on to move towards the opposite - default no access, and subtypes for horizon. This is cause we have so much more offships and away sites that use everything more than horizon.

File: code/game/objects/structures/crates_lockers/closets/secure/bar.dm [hunk 5]
```diff
@@ -23,3 +23,26 @@
...
+	new /obj/item/storage/box/fancy/yoke/ebisu( src )
+	new /obj/item/storage/box/fancy/yoke/ebisu( src )
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
```
Comment (SleepyGemmy) [inline code/game/objects/structures/crates_lockers/closets/secure/bar.dm:45]:
```suggestion
	new /obj/item/storage/box/fancy/yoke/beer(src)
	new /obj/item/storage/box/fancy/yoke/beer(src)
	new /obj/item/storage/box/fancy/yoke/ebisu(src)
	new /obj/item/storage/box/fancy/yoke/ebisu(src)
	new /obj/item/storage/box/fancy/yoke/shimauma(src)
	new /obj/item/storage/box/fancy/yoke/shimauma(src)
	new /obj/item/storage/box/fancy/yoke/moonlabor(src)
	new /obj/item/storage/box/fancy/yoke/moonlabor(src)
	new /obj/item/storage/box/fancy/yoke/earthmover(src)
	new /obj/item/storage/box/fancy/yoke/earthmover(src)
	new /obj/item/storage/box/fancy/yoke/whistlingforest(src)
	new /obj/item/storage/box/fancy/yoke/whistlingforest(src)
```
i assume that you followed the booze closet example. if you could fix that as well i'd appreciate it, but you don't have to of course.

**Review by DreamySkrell — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**